mkosi

```diff
@@ -85,6 +85,8 @@ class CommandLineArguments(argparse.Namespace):
 
     swap_partno = None  # type: Optional[int]
     esp_partno = None  # type: Optional[int]
+    with_network = None  # type: Optional[bool]
+    with_docs = None  # type: Optional[bool]
```

Comment:
@keszybz is this the preferred way to fix? Or should we just pass `default=None` to the option?

Reply:
I think default=None is preferable.